recipes/tetrimmer/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "TEtrimmer" %}
+{% set version = "1.3.0" %}
+{% set sha256 = "2d9f7c615fb289372ef8e005bf44d951467798928104605c2aff2b8ce93edf26" %} 
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  # url: https://github.com/qjiangzhao/TEtrimmer/releases/download/v{{ version }}/tetrimmer-{{ version }}.tar.gz
+  url: https://github.com/qjiangzhao/TEtrimmer/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('tetrimmer', max_pin="x") }}
+
+requirements:
+  run:
+    - python
+    - numpy
+    - perl 
+    - r-base
+    - blast
+    - biopython
+    - bedtools
+    - cd-hit
+    - dataclasses
+    - emboss
+    - hmmer
+    - iqtree
+    - matplotlib-base
+    - mafft
+    - multiprocess
+    - pandas
+    - pfam_scan
+    - plotly
+    - pypdf2
+    - regex
+    - repeatmodeler  
+    - repeatmasker
+    - requests
+    - samtools
+    - scikit-learn
+    - tk
+    - urllib3
+    - click
+    - dill
+    - joblib
+    - nseg
+    - recon
+    - trf
+    - ghostscript
+
+test:
+  commands:
+    - TEtrimmer --help
+    # - python TEtrimmer-main/tetrimmer/TEtrimmer.py --help
+    
+about:
+  home: https://github.com/qjiangzhao/TETrimmer.git
+  license: GPL-3
+  summary: "TETrimmer is designed to replace and assist TE manual curation."
+  license_family: GPL
```

Comment:
I think you will need to add `license_file: ...` too. It should point to the file in the source tarball.